platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/AddressBookInitializer.java

```diff
@@ -136,7 +136,7 @@ public AddressBook getInitialAddressBook() {
 
     /**
      * Determines the address book to use.  If the configured current version of the application is higher than the save
-     * state version, the swirld state is given the config address book to determine stake weights.  If the address book
+     * state version, the swirld state is given the config address book to determine weight weights.  If the address book
```

Comment:
`weight weights` doubles the object. Should be `weights`

Reply:
deleting the first weight